snmp/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,403 @@
+name: SNMP
+files:
+- name: snmp.yaml
+  options:
+    - template: init_config
+      options:
+      - name: loader
+        required: false
```

Comment:
Is `required: false` the default? if so, maybe we can omit it ?

Reply:
Yes! it is the default! I will change it